packages/core/core/src/parcel-v3/Parcel.js

```diff
@@ -44,6 +47,15 @@ export class ParcelV3 {
       // Ping
       case 0:
         return;
+      case 1:
```

Comment:
Consider whether this is strictly better than the multi-threaded func. delegation on `file_system_napi.rs` ; 

One is centralized in one file, while the other can interface with any JS object.

Reply:
Yeah, I'm finding these changes hard to grok